src/acl/ConnMark.cc

```diff
@@ -15,6 +15,7 @@
 #include "Debug.h"
 #include "http/Stream.h"
 #include "sbuf/Stream.h"
+#include <limits>
```

Comment:
Please double check whether this header is needed. I do not see any `std` names in this file.